### PR TITLE
update SQLX descriptions for Theta

### PR DESCRIPTION
**Pull request by leerho**
This is my first attempt to contribute to this repo.  I copied the style from the G-BQ SQLX functions to some extent.
Nonetheless, I have a few style consistency questions below.

### CODE REVIEW

File: theta_sketch_agg_union.sql [hunk 1]
```diff
@@ -15,8 +15,17 @@
 # specific language governing permissions and limitations
 # under the License.
 
-CREATE OR REPLACE AGGREGATE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_agg_union(sketch BYTES, lg_k INT64 NOT AGGREGATE) RETURNS BYTES LANGUAGE js
-OPTIONS (library=["$GCS_BUCKET/theta_sketch.mjs"]) AS R"""
+CREATE OR REPLACE AGGREGATE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_agg_union(sketch BYTES, lg_k INT64 NOT AGGREGATE)
```
Comment (leerho) [inline theta_sketch_agg_union.sql:18]: This is the only one that didn't require a seed.  Is this correct?

File: theta_sketch_a_not_b.sql [hunk 2]
```diff
@@ -15,8 +15,18 @@
...
-CREATE OR REPLACE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_a_not_b(sketch1 BYTES, sketch2 BYTES, seed INT64) RETURNS BYTES LANGUAGE js
-OPTIONS (library=["$GCS_BUCKET/theta_sketch.js"]) AS R"""
+CREATE OR REPLACE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_a_not_b(sketch1 BYTES, sketch2 BYTES, seed INT64)
+RETURNS BYTES
+LANGUAGE js
+OPTIONS (
+  library=["$GCS_BUCKET/theta_sketch.js"],
+  description = '''Performs the scalar set difference: sketch1 and not sketch2.
+Param sketch1: the first sketch "a" as bytes.
+Param sketch2: the second sketch "b" as bytes.
```
Comment (leerho) [inline theta_sketch_a_not_b.sql:25]: I think we need to be consistent and clear just what kind of sketch "bytes" we are expecting: are these Base64 encoded bytes, or raw bytes?  Do they represent a Compressed Compact Sketch, always?

File: theta_sketch_scalar_intersection.sql [hunk 3]
```diff
@@ -15,8 +15,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-CREATE OR REPLACE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_scalar_intersection(sketchBytes1 BYTES, sketchBytes2 BYTES, seed INT64) RETURNS BYTES LANGUAGE js
-OPTIONS (library=["$GCS_BUCKET/theta_sketch.js"]) AS R"""
+CREATE OR REPLACE FUNCTION `$BQ_PROJECT.$BQ_DATASET`.theta_sketch_scalar_intersection(sketchBytes1 BYTES, sketchBytes2 BYTES, seed INT64)
```
Comment (leerho) [inline theta_sketch_scalar_intersection.sql:18]: could we not use "sketch1" and "sketch2" as in the other set operations?

**Comment by jmalkin** [comment]
Did you make sure these changes are valid for .sql files? We have .sql, not .sqlx.

**Comment by leerho** [comment]
G-BQ uses .sqlx for the file type.  And this code is destined to be put alongside the other SQLX functions that the G-BQ team has already developed.  

The use of .sql as the file type is not incorrect, per-se. .sqlx is a pure extension of .sql that allows additional documentation fields and other metadata.  So any proper .sql file is also a proper .sqlx file.

However, we should be following the example of the functions that G-BQ folks designed using .sqlx.   Why wouldn't we want our SQL functions documented for ease of use by the users?

Our current .sql files should be renamed as .sqlx.

File: theta_sketch_agg_union.sql (hunk 1 above)
Comment (AlexanderSaydakov) [reply theta_sketch_agg_union.sql:18]: not implemented yet. I will do that shortly

File: theta_sketch_a_not_b.sql (hunk 2 above)
Comment (AlexanderSaydakov) [reply theta_sketch_a_not_b.sql:25]: as I understand, base64 is implicitly done by the engine. from the user's perspective this is a binary blob. Functions don't not care if the input sketches are compressed or not. We may specify if the output is compressed or not, and maybe even make it configurable, but I am inclined to think that more CPU to reduce IO must be an almost universally good trade-off in systems like this.

File: theta_sketch_scalar_intersection.sql (hunk 3 above)
Comment (AlexanderSaydakov) [reply theta_sketch_scalar_intersection.sql:18]: we could. feel free to improve consistency

**Comment by AlexanderSaydakov** [comment]
We may want to rename to sqlx, but I don't think that adding description requires this.

**Comment by AlexanderSaydakov** [comment]
It seems that descriptions are not implemented in BigQuery.
> describe t.theta_sketch_get_estimate;
Statement not supported: DescribeStatement at [1:1]

**Comment by leerho** [comment]
There is more consistency issues to fix, but because of the change in file suffix, we need to check in what we have.  So far, the only changes are adding the "description" field to the SQL declaration and the file suffix.

**Review by AlexanderSaydakov — APPROVED** [review]

**Pull request merged**